lib/tic_tac_toe_api_web/controllers/page_controller.ex

```diff
@@ -1,22 +1,38 @@
 defmodule TicTacToeApiWeb.PageController do
   use TicTacToeApiWeb, :controller
 
+  @env Application.get_env(:tic_tac_toe_api, :env)
+
   def index(conn, _params) do
     board = %Board{}
     render(conn, "index.json", board: board)
   end
 
   def status(conn, %{"spots" => spots, "next_player" => next_player, "current_player" => current_player}) do
-    result = build_result(spots, next_player, current_player)
+    result = board_with_players(spots, next_player, current_player)
+             |> Status.result(next_player, current_player)
     render(conn, "status.json", result: result)
   end
 
-  defp build_result(spots, next_player, current_player) do
-    player1 = %Player{symbol: next_player}
-    player2 = %Player{symbol: current_player}
-    spots_list = Jason.decode!(spots)
-                 |> ReactAdapter.swap_marks_with_players(player1, player2)
-    Status.result(spots_list, next_player, current_player)
+  defp board_with_players(spots, player1, player2) do
+    player1 = %Player{symbol: player1}
+    player2 = %Player{symbol: player2}
+    Jason.decode!(spots)
+    |> ReactAdapter.swap_marks_with_players(player1, player2)
+  end
+
+  def computer(conn, %{"spots" => spots, "computer" => computer, "human" => human}) do
+    move = board_with_players(spots, computer, human)
+           |> get_computer_move(@env)
+
+    render(conn, "computer.json", move: move)
   end
 
+  defp get_computer_move(board, :test) do
```

Comment:
I like the fact that you can test this function. But maybe inject the seed rather than the environment?